lib/msfl_visitors/renderers/chewy_renderer.rb

```diff
@@ -0,0 +1,41 @@
+module MSFLVisitors
+  module Renderers
+    class ChewyRenderer
+      BINARY_OPERATORS = {
```

Comment:
Not sure what you mean - anything can be a hash key, in this case I'm using the class constants as the keys, because it makes the mapping easier.